frontend/doc-site/032__Contribute and Publish Data.mdx

```diff
@@ -1,105 +1,141 @@
 # Contributing Data
 
-CELLxGENE supports a rapidly growing single cell data corpus because of generous contributions from researchers like you! If you have single cell data that meet our [requirements](#dataset-requirements), please reach out to our curation team at [cellxgene@chanzuckerberg.com](mailto:cellxgene@chanzuckerberg.com).
+CELLxGENE supports a rapidly growing single cell data corpus because of generous contributions from researchers like you! Please see below for If you have single-cell data that meet our [requirements](#dataset-requirements), please reach out to our curation team at [cellxgene@chanzuckerberg.com](mailto:cellxgene@chanzuckerberg.com).
 
 ## Publishing Process
 
-The process for submission to the portal is:
-
-- You confirm we can support your submission by reaching out to our curation team at [cellxgene@chanzuckerberg.com](mailto:cellxgene@chanzuckerberg.com) with a description of the data that you'd like to contribute.
-- We confirm that we will accept your data.
-- You prepare your data according to our submission [requirements](#dataset-requirements) and send us your files.
-- We upload to a private Collection where you can review.
-- You prepare revised data and send us your revised files, as needed.
-- We publish the data when you tell us to.
+1. To confirm we can support your submission, please [contact us] with a description of the data that you'd like to contribute
+2. We confirm that we will accept your data
+3. You send us files prepared according to the submission [requirements](#dataset-requirements)
+4. We upload to a private Collection where you can review
+5. The submission can be revised, as needed
+6. The data are made openly available when you are ready
 
 ## Dataset Requirements
 
 ### Data Eligibility
 
-CELLxGENE is focused on supporting the global community attempting to create references of human cells and tissues. As a result, there are a few kinds of datasets that we are **not** likely to accept at this time:
+CELLxGENE supports most single-cell RNA-seq and ATAC-seq data, but there are a few types of data that we are **not** likely to accept at this time:
 
 - drug screens
 - cell lines
-- organisms other than mouse or human
-- assays not on the [CELLxGENE Census accepted assays list](https://github.com/chanzuckerberg/cellxgene-census/blob/main/docs/census_accepted_assays.csv)
-  - These additional assays are pending Census acceptance and will be accepted:
-    - Visium (non-HD)
-    - Slide-seq
-    - Expression measurements of multi-modal assays (e.g. 10x multiome, mCT-seq)
+- species not on the [supported list](https://chanzuckerberg.github.io/single-cell-curation/latest-schema.html#organism_ontology_term_id)
+- assays not on the [supported list](https://github.com/chanzuckerberg/cellxgene-census/blob/main/docs/census_accepted_assays.csv)
```

Comment:
@ivirshup - is there a regular process to review/update this CSV during a census schema update?

Reply:
Not that I know of. We even seem to have missed it [here](https://github.com/chanzuckerberg/cellxgene-census/pull/1304), and the commit history for that file is short.

The functional list of allowed assays lives here: https://github.com/chanzuckerberg/cellxgene-census/blob/bfb40f6acc5b12f31aa24a2f8f244355e876d779/tools/cellxgene_census_builder/src/cellxgene_census_builder/build_soma/globals.py#L211-L325

@ebezzi do you know the purpose of this file?

Reply:
@jahilton - any concerns about referencing the code (similar to the consortia case). It does not make sense to have multiple sources of truth.

Reply:
The code doesn't really fit this use case very well. A standalone, searchable table provides a much better user experience.
But fully agree that 1 source of truth is the goal, but that 1 source can have derived products automatically generated from it to serve additional use cases.

Reply:
The csv covers nearly all assays that are in the code - only missing Parse terms, 10x flex, STORM-seq, 2 variations of STRT-seq (though includes others), tag based single cell RNA sequencing.
So I propose we continue linking to the csv & update once we have a more reliable source.

Reply:
I am fine with that, and can update the csv's. It looks like we also link to this from the census schema

Reply:
I've opened as issue to track here: https://github.com/chanzuckerberg/cellxgene-census/issues/1388

Reply:
The csv is now up-to-date
TY @ivirshup !